src/utils/swapMinAmounts.ts

```diff
@@ -90,43 +109,70 @@ export async function swapMinAmounts(
       })
   }
 
-  // Enable all swap plugins
-  for (const swapPluginConfig of Object.values(account.swapConfig)) {
-    await swapPluginConfig.changeEnabled(true)
-  }
+  const allMinAmtsObj: { [key: string]: SingleCurrencyPairMinAmt } = {}
 
-  // Initial parameters for findMinSwapAmount
-  // NOTE: This is a placeholder that will be replaced in the future
-  const initBinarySearchMin = 0
-  const initBinarySearchMax = 1000000
-  const fromCurrencyWallet = currencyWallets[0]
-  const toCurrencyWallet = currencyWallets[1]
-
-  // Get an array of all the swap plugin names
-  const swapPluginNamesArr = Object.keys(account.swapConfig)
-
-  // Create a variable to store the swap quotes with the plugin name as the key
-  const minAmountObj: { [key: string]: number } = {}
-
-  // Loop over each plugin to attempt to get a swap quote
-  for (const plugin of swapPluginNamesArr) {
-    try {
-      const disabledPluginMap = createDisabledSwapPluginMap(
-        plugin,
-        swapPluginNamesArr
-      )
-      const requestOptions = { disabled: disabledPluginMap }
-      minAmountObj[plugin] = await findMinSwapAmount(
-        initBinarySearchMin,
-        initBinarySearchMax,
-        fromCurrencyWallet,
-        toCurrencyWallet,
-        requestOptions
-      )
-    } catch (e) {
-      console.log(e)
+  try {
+    // Enable all swap plugins
+    for (const swapPluginConfig of Object.values(account.swapConfig)) {
+      await swapPluginConfig.changeEnabled(true)
     }
-  }
 
-  return minAmountObj
+    // Get an array of all the swap plugin names
+    const swapPluginNamesArr = Object.keys(account.swapConfig)
```

Comment:
You could move this out of the for loop, and use it in the for loop creation:
for  (const swapPluginConfig of swapPluginNamesArr)